.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -85,7 +87,8 @@ body:
   - type: dropdown
     id: os
     attributes:
-      label: What Operating System are you seeing the problem on?
+      label: Relevant OSes
```

Comment:
**suggestion**: I think the correct pluralization here would be `OSs`, might be wrong tho